### PR TITLE
Update with a new fission safety, correct a typo in the djinn guide

### PR DESCRIPTION
**Pull request by atlgeek007**
:)

### CODE REVIEW

File: gameplay/how-to.../occultism-automatic-djinn-repair.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Not with the power of several mods in Enigmatica 6, that is!  Using a combinatio
 # What you will need:
 
 - A working knowledge of how Pedestals works.
-- An "XP to Fluid" converter pedestal
+- An "Fluid to XP" converter pedestal
```
Comment (NielsPilgaard) [inline gameplay/how-to.../occultism-automatic-djinn-repair.md:15]: `An` -> `A` now that they're reversed :D

File: gameplay/how-to.../mekanism-simple-fission-safety.md [hunk 2]
```diff
@@ -49,4 +49,26 @@ And turn it on...
 
 My reactor would have exploded! But the safety breaker turned the reactor off. This setup should be able to handle very large heating rates.
 
+## A slightly more advanced safety
```
Comment (NielsPilgaard) [inline gameplay/how-to.../mekanism-simple-fission-safety.md:52]: `A slightly more advanced safety` -> `Slightly more advanced safety` is more natural sounding imo :)

**Review by NielsPilgaard — APPROVED** [review]

**Comment by atlgeek007** [comment]
Ugh, English is dumb ;)  Fixed.

**Review by NielsPilgaard — APPROVED** [review]
Good stuff, thanks!

**Pull request merged**